docs/rackhd/install_os.rst

```diff
@@ -68,7 +69,7 @@ All parameters descriptions of OS installation workflow payload are listed below
 Parameters          Type              Flags       Description
 =================== ================ ============ ============================================
 version             String           **required** The version number of target OS that needs to install. **NOTE**: For Ubuntu, *version* should be the codename, not numbers, for example, it should be "trusty", not "14.04"
-repo                String           **required** The OS repository address, currently only supports HTTP. Some examples of free OS distributions for reference. For **CentOS**, http://mirror.centos.org/centos/7/os/x86_64/. For **Ubuntu**, http://us.archive.ubuntu.com/ubuntu/. For **openSUSE**, http://download.opensuse.org/distribution/leap/42.1/repo/oss/. For enterprise OS distributions like **ESXi**, **RHEL**, **SLES**, the repository is the directory of mounted DVD ISO image, and http service is provided for this directory. 
+repo                String           **required** The OS repository address, currently only supports HTTP. Some examples of free OS distributions for reference. For **CentOS**, http://mirror.centos.org/centos/7/os/x86_64/. For **Ubuntu**, http://us.archive.ubuntu.com/ubuntu/. For **openSUSE**, http://download.opensuse.org/distribution/leap/42.1/repo/oss/. For enterprise OS distributions like **ESXi**, **RHEL**, **SLES**, the repository is the directory of mounted DVD ISO image, and http service is provided for this directory.
```

Comment:
curious what change in thin line :), I read words one by one, no change found

Maybe 'currently only supports HTTP' is not accurate now.  or 'currently non-windows OS only supports HTTP'? or because windows payloads are very different with other OSes. just change this section to 'Non-Windows OS Installation Workflow Payload' ?